src/openforms/authentication/api/views.py

```diff
@@ -45,5 +45,10 @@ class AuthenticationLogoutView(APIView):
         ),
     )
     def delete(self, request, *args, **kwargs):
+        for plugin in register.iter_enabled_plugins():
+            logout_method = getattr(plugin, "logout", None)
+            if logout_method and callable(logout_method):
+                logout_method(request)
+
```

Comment:
Let's add a default no-op method to the Base plugin instead, which can/should be overridden and stipulates a clear contract. Then you can get rid of the `getattr` call and if/else checking, but just always call it - the default then does nothing.